### PR TITLE
Information Architecture v3

### PR DESCRIPTION
**Pull request by mDuo13**
This continues the work from the IA.v2 branch (PR #1592) but addresses issues that came up in reconciling changes with the master branch

### CODE REVIEW

**Comment by mDuo13** [comment]
Rebased and force-pushed to fix the merge conflict. Hopefully this will be the last.

**Comment by tequdev** [comment]
The following file isn't used anywhere in `dactyl-config.yml`
`content/concepts/introduction/xrp.md`

---
this will resolve
- https://github.com/XRPLF/xrpl-dev-portal/pull/2039

File: content/concepts/introduction/what-is-xrp.md
Comment (Silkjaer) [inline content/concepts/introduction/what-is-xrp.md]: This should be elaborated. XRP is held in escrow, but released to whom and is the sales only to help maintain the stability of the currency?

File: content/concepts/introduction/what-is-xrp.md
Comment (Silkjaer) [inline content/concepts/introduction/what-is-xrp.md:58]: Should add a paragraph about trademarks.

**Review by Silkjaer — CHANGES_REQUESTED** [review]
Thanks for this great update, it's overall a fantastic rearrangement and additions to the already good documentation. I added a few notes, and have shared preview links with a few others to get some external feedback that i'll be adding in a few days (if any).

File: content/contributing/documentation-languages.md
Comment (tequdev) [inline content/contributing/documentation-languages.md]: When this PR is merged into the master branch, `content/concepts/introduction/xrp-ledger-overview.md` will not exist in the branch.

**Review by tequdev — CHANGES_REQUESTED** [review]

**Comment by mDuo13** [comment]
@develoQ I believe the changes you requested have been addressed by #2091.
@Silkjaer I believe the changes you requested have been addressed by #2099.

**Review by tequdev — CHANGES_REQUESTED** [review]
スクリーンショット 2023-09-01 13 28 30

Error logs are being output at build time that were not there before.

 I will now check other Japanese-related sections, including #2091.

**Review by Silkjaer — APPROVED** [review]
LGTM content-wise and ready to merge if the issues @develoQ mentioned are fixed.

**Review by tequdev — APPROVED** [review]
There were no serious problems with the Japanese pages.

I have made some fixes to improve side-nav,top-nav for japanese.

If this PR will be merged into the main branch soon, I'll create a PR for main branch after the merge.
If it will not be merged for a while, I will create a PR targeting IA.v3 branch.

**Comment by amarantha-k** [comment]
Yes, we plan to merge soon. 
+1 to creating a PR for the main branch after the merge. 
Thanks for your contribution!

**Pull request merged**
